docs/README.md

```diff
@@ -18,6 +18,9 @@
     - [Copiable](copiable.md)
         - [Generating Copiable Methods](copiable.md#generating-copiable-methods)
         - [Modifying The Copiable Code Generation](copiable.md#modifying-the-copiable-code-generation)
+    - [Fakeable](fakeable.md)
+        - [Generating Fake Methods](fakeable.md#generating-fake-methods)
+        - [Modifying Fakes Code Generation](fakeable.md#modifying-the-fakeable-code-generation)
```

Comment:
Let's also add these to the **main README** page → https://github.com/woocommerce/woocommerce-ios/tree/issue/3096-documentation#-documentation. 

I think we should just remove this `docs/README.md` file since it's repetitive. Or at least change `docs/README.md` to say “The index is at `/README.md`”.  I could not decide during the time I created this file. 😅

Reply:
Ha I totally missed that the content was repeated, I also think deleting this file is better!